tests/testthat/test-tags.r

```diff
@@ -893,3 +893,19 @@ test_that("extractPreserveChunks works for emoji strings", {
     c('chunk2', 'chunk1')
   )
 })
+
+
+test_that("tag attributes are normalized on creation", {
+  d <- div(class = "a", "text1", class = "b c", "text2")
```

Comment:
Can you also test when a vector is used, like `c("d", "e")`? See #205.

Reply:
a0dfdeb